### PR TITLE
Naming and version changes

### PR DESCRIPTION
**Pull request by dethos**
This pull request addresses 3 issues:

* It tries to uniformize the name of the project
* Removes the beta disclaimer, since it will be marked as version 1.0 soon
* Fixes some formatting and spacing issues on 2 templates

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Contributing to Hawkpost
 
-Given this is an unfinished project and many features are still missing, you are more than welcome to join in and help improve HawkPost. The project will mostly use the Github issues to keep track of bugs, feature requests and milestones. So and account should be all you need to start contributing.
+Given this is an unfinished project and many features are still missing, you are more than welcome to join in and help improve Hawkpost. The project will mostly use the Github issues to keep track of bugs, feature requests and milestones. So and account should be all you need to start contributing.
```
Comment (guilhermesimoes) [inline CONTRIBUTING.md]: Maybe reword this first sentence since you are removing the beta disclaimer. Congrats! 🎆

Comment (dethos) [reply CONTRIBUTING.md]: Done, thanks.

**Review by jlbribeiro — APPROVED** [review]
LGTM 👍
